CONTRIBUTING.rst

```diff
@@ -75,7 +75,7 @@ Ready to contribute? Here's how to set up `bluesky-queueserver` for local develo
 
 5. When you're done making changes, check that your changes pass flake8 and the tests, including testing other Python versions with tox::
 
-    $ flake8 bluesky_queueserver tests
+    $ flake8 bluesky_queueserver/tests
```

Comment:
This file came from the templates via https://github.com/NSLS-II/scientific-python-cookiecutter. I think, historically some projects had the `tests` directory on the same level as the library with modules (`bluesky_queueserver/` in this case). As the directories with the tests are a part of the library now, this can be as simple as:
```suggestion
    $ flake8
```
There is a file `.flake8` in the root of the repo, which "controls" the flake8's behavior and operates the same way in [CI](https://github.com/bluesky/bluesky-queueserver/blob/e3a0fa0ab98ec6eb31bc7b1466b6b07bfa63badf/.github/workflows/flake8.yml#L21).